portfolio/src/main/webapp/blog_script.js

```diff
@@ -13,35 +13,48 @@
 // limitations under the License.
 
 
+
 function getComments(){
+    var currUserEmail = null; 
+    fetch('/login').then(response => response.json()).then((userInfo) =>{
+        currUserEmail = userInfo[1];
+        isAdmin = userInfo[2];
+    });
     fetch('/comments').then(response => response.json()).then((comments) =>{
         comments.forEach((comment) => {
             const commentHistory = document.getElementById(comment.postId);
-            commentHistory.appendChild(createCommentElement(comment));
+            commentHistory.appendChild(createCommentElement(comment, currUserEmail));
         })
     });
 }
 
-function createCommentElement(comment){
+function createCommentElement(comment, currUserEmail){
     const commentElement = document.createElement('li');
     commentElement.className = 'comment';
 
     const textElement = document.createElement('p');
     textElement.innerText = comment.text;
 
-    const dateElement = document.createElement('h5');
+    const dateElement = document.createElement('h6');
     dateElement.innerText = comment.date;
 
-    const deleteButtonElement = document.createElement('button');
-    deleteButtonElement.innerText = 'Delete';
-    deleteButtonElement.addEventListener('click', () => {
-        deleteComment(comment);
-        commentElement.remove();
-    });
+    const userEmailElement = document.createElement('h5');
+    userEmailElement.innerText = comment.userEmail;
 
+    commentElement.appendChild(userEmailElement);
     commentElement.appendChild(dateElement);
     commentElement.appendChild(textElement);
-    commentElement.appendChild(deleteButtonElement);
+
+    if(currUserEmail == comment.userEmail || isAdmin == "true"){
```

Comment:
Could you apply formatter please. (Spaces between if/else)

Reply:
done.